.github/workflows/kind-e2e.yaml

```diff
@@ -118,15 +114,25 @@ jobs:
 
           wait_until_pods_running kafka && wait_until_pods_running knative-eventing
 
-      - name: Run e2e Tests
+      - name: Run e2e Tests (./test/e2e)
+        run: |
+          export KO_FLAGS="--platform=linux/amd64"
+          export SYSTEM_NAMESPACE=knative-eventing
+          export CLUSTER_DOMAIN=${CLUSTER_SUFFIX}
+
+          # Run the tests tagged as e2e on the KinD cluster.
+          go test -race -count=1 -timeout=1h -v -short -tags=e2e \
+             ./test/e2e ${{ matrix.extra-test-flags }} -channels=messaging.knative.dev/v1beta1:KafkaChannel
+
+      - name: Run e2e Tests (./test/e2e_new)
```

Comment:
We decided to split them so that they run in parallel and it's easier to find the problem in GH actions, why run them sequentially?

We can use the `extra-test-flags` parameter to specify the `-channels=..` parameters with something like:

```yaml
include:
  ...
  - test-suite: ./test/e2e
  - extra-test-flags: -channels=messaging.knative.dev/v1beta1:KafkaChannel
```

Reply:
Let me do this change when we see tests passing